src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicenceIntegrationTest.kt

```diff
@@ -245,6 +248,30 @@ class LicenceIntegrationTest : IntegrationTestBase() {
     assertThat(result?.submittedByFullName).isEqualTo("Test Client")
   }
 
+  @Test
+  @Sql(
+    "classpath:test_data/seed-prrd-licence-id-1.sql",
+  )
+  fun `Submit PRRD licence`() {
+    // Given
+    prisonerSearchApiMockServer.stubSearchPrisonersByNomisIds()
+
+    // When
+    webTestClient.put()
+      .uri("/licence/id/1/submit")
+      .accept(MediaType.APPLICATION_JSON)
+      .headers(setAuthorisation(roles = listOf("ROLE_CVL_ADMIN")))
+      .exchange()
+      .expectStatus().isOk
+
+    // Then
+    val licence = licenceRepository.findById(1).getOrNull()
+    assertThat(licence).isNotNull
+    assertThat(licence!!).isInstanceOf(PrrdLicence::class.java)
+    assertThat(licence.kind).isEqualTo(LicenceKind.PRRD)
+    assertThat(licence.postRecallReleaseDate).isNotNull()
+  }
```

Comment:
This should probably test to make sure that the licence status changes to `Submitted`, though there's a chance that it won't at the moment as we perform an eligibility check on submission and recalls are still ineligible 🤔